pandas/core/arrays/datetimes.py

```diff
@@ -570,7 +570,7 @@ def dtype(self) -> np.dtype | DatetimeTZDtype:  # type: ignore[override]
     @property
     def tz(self) -> tzinfo | None:
         """
-        Return timezone, if any.
+        Return the timezone.
```

Comment:
why no "if any"?

Reply:
IMO it sounded better with `if any` (if it returns `None` that's the timezone). Can change it back if you feel strongly about it.

Reply:
> if you feel strongly about it.

nope.